iac-extensions/docker/src/test/resources/checks/PackageManagerUpdateWithoutInstallCheck/PackageManagerUpdateWithoutInstallCheck.dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM scratch
+
+# Noncompliant@+1 {{Update cache and install packages in single RUN instruction.}}
+RUN apk update
+#   ^^^^^^^^^^
+# Noncompliant@+1
+RUN apt update
+# Noncompliant@+1
+RUN apt-get update
+# Noncompliant@+1
+RUN aptitude update
+# Noncompliant@+1
+RUN apt-get update && echo APT index updated
+# Noncompliant@+1
+RUN echo Preparing to apt-get install && apt-get update
+
+# Noncompliant@+2
+RUN <<EOF
+apt-get update
+EOF
+# Noncompliant@+3
+RUN <<EOF
+echo Preparing to apt-get install
+apt-get update
+EOF
+# Noncompliant@+2
+RUN <<EOF
+apt-get update
+echo APT index updated
+EOF
+
+# Complaiant
+RUN apk update && apk add libcurl
+RUN apt update && apt install libcurl
+RUN apt-get update && apt-get install libcurl
+RUN aptitude update && aptitude install libcurl
+RUN aptitude update && apt-get install libcurl
```

Comment:
Is `aptitude` and `apt-get` using the same cache?
Shouldn't `apt` then use it to? If yes I would add
`RUN apt update && apt-get install libcurl` or some other variations

Reply:
Yes, both `apt` and `aptitude` are frontends to `apt-get`. I didn't incorporate any particular checks on whether the same cache is used; I thought that the probability of seeing `apk` and `apt-get` being mixed in a single `RUN` instruction is too low to increase the rule's complexity for it. I'll add some more variations to better illustrate it in tests.